scss/_popover.scss

```diff
@@ -57,18 +58,17 @@
 
     .arrow::before,
     .arrow::after {
+      top: 100%;
```

Comment:
popper.js manages `.arrow` not `.arrow:before` and `.arrow:after`.

So it's safe to position these pseudo-elements.